crates/auth/src/identity.rs

```diff
@@ -49,39 +47,33 @@ pub struct IncomingClaims {
 }
 
 impl TryInto<SpacetimeIdentityClaims> for IncomingClaims {
-    type Error = TokenValidationError;
+    type Error = anyhow::Error;
 
```

Comment:
Why was removed the `TokenValidationError`?

Reply:
This method never actually returned anything other than its `Other(anyhow::Error)` variant, so it's easier to just return `anyhow::Error`.